scripts/deployWeb.sh

```diff
@@ -12,12 +12,9 @@ else
 fi
 
 echo "========= Downloading the data =========="
-if [ -e allData.tgz ]; then
-  rm allData.tgz
-fi
-aria2c  -l "" https://replicability.graphics/allData.tgz
+git clone https://github.com/GraphicsReplicability/replicability.graphics.git tmp
```

Comment:
```suggestion
if [ ! -d tmp ]; then
    if [ ! -d .git ]; then
        rm -rf ./*
    fi
    git clone --depth 1 https://github.com/GraphicsReplicability/replicability.graphics.git tmp
fi
```

Reply:
to be tested.

Reply:
ok for the clean but not for the `--depth 1` (I tried that but I cannot checkout the gh-pages branch with this)

Reply:
maybe add `-b gh-pages`, and remove the checkout command below (replace by pull) ?